zulip/zulip/__init__.py

```diff
@@ -601,16 +608,21 @@ def end_error_retry(succeeded: bool) -> None:
                     "status_code": res.status_code}
 
     def call_endpoint(self, url: Optional[str] = None, method: str = "POST", request: Optional[Dict[str, Any]] = None,
-                      longpolling: bool = False, files: Optional[List[IO[Any]]] = None, timeout: Optional[float] = None) -> Dict[str, Any]:
+                      longpolling: bool = False, files: Optional[List[IO[Any]]] = None, timeout: Optional[float] = None,
+                      accessVersionedAPI: bool = True) -> Dict[str, Any]:
         if request is None:
             request = dict()
         marshalled_request = {}
         for (k, v) in request.items():
             if v is not None:
                 marshalled_request[k] = v
-        versioned_url = API_VERSTRING + (url if url is not None else "")
-        return self.do_api_query(marshalled_request, versioned_url, method=method,
-                                 longpolling=longpolling, files=files, timeout=timeout)
+        effective_url = (url if url is not None else "")
+        accesswithAPI = False
+        if accessVersionedAPI:
+            effective_url = API_VERSTRING + effective_url
+            accesswithAPI = True
+        return self.do_api_query(marshalled_request, effective_url, method=method,
+                                 longpolling=longpolling, files=files, timeout=timeout, accesswithAPI = accesswithAPI)
```

Comment:
I think `accesswithAPI = accessVersionedAPI` makes more sense, we don't need a separate variable `accesswithAPI`.